CONTRIBUTING.md

```diff
@@ -176,7 +176,7 @@ fits in based on what it does.
     | 
     |           Generic utility functions and miscellaneous tools.
     |           The contents of this submodule should be general enough to be 
-    |           useable outside of OpenHTF, meaning it should not be dependent 
+    |           usable outside of OpenHTF, meaning it should not be dependent
```

Comment:
Discussion: https://english.stackexchange.com/questions/69709/is-useable-preferred-in-certain-regions-or-just-an-alternate-spelling-of-usa